.env.example

```diff
@@ -0,0 +1,6 @@
+DB_PORT=
+DB_HOST=
+HOST_PORT=
+MONGO_URI=
+NODE_ENV=
```

Comment:
This is something that is absolutely irrelevant to have in a `.env` file, as it is set either by `node`, `npm` or some docker configs depending on the context. It is used as an environment variable directly and thus has no place in this file.